x-pack/test/security_solution_api_integration/test_suites/detections_response/rules_management/rule_import_export/trial_license_complete_tier/import_rules_with_overwrite.ts

```diff
@@ -0,0 +1,191 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from 'expect';
+
+import { DETECTION_ENGINE_RULES_URL } from '@kbn/security-solution-plugin/common/constants';
+import { createRule, deleteAllRules } from '../../../../../../common/utils/security_solution';
+import { combineToNdJson, getCustomQueryRuleParams, fetchRule } from '../../../utils';
+import { FtrProviderContext } from '../../../../../ftr_provider_context';
+
+export default ({ getService }: FtrProviderContext): void => {
+  const supertest = getService('supertest');
+  const log = getService('log');
+
+  describe('@ess @brokenInServerless @skipInQA import_rules with rule overwrite set to "true"', () => {
```

Comment:
Why `brokenInServerless`?

Reply:
Good point. I left the tests disabled in Serverless as `trial_license_complete_tier/import_rules.ts` is disabled in Serveless. After checking locally I unskipped `import_rules_with_overwrite.ts` for Serverless because tests pass.